<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newlogin</class>
 <widget class="QDialog" name="newlogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Please fill the following information. All fields are required to create a new account.</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>761</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>New Account</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="newbakButton">
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>49</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;- BACK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newcreButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>49</height>
        </size>
       </property>
       <property name="text">
        <string>CREATE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newcanButton">
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>49</height>
        </size>
       </property>
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>721</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>                      First Name :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>                     Last Name :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>              Address Line 1 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>              Address Line 2 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>                                City :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>                              State :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>                        Zip Code :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>                             Phone :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>                            D.O.B. :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>                          Login ID :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>             Login Password :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Login Password Confirm :</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="firstNameEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lastNameEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="address1Edit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="address2Edit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="cityEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="stateEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="zipCodeEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="phoneEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="dobEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="loginIdEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="loginPassEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="loginPassConfirmEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>newbakButton</zorder>
   <zorder></zorder>
   <zorder></zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
